textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/planstore/PlanStoreConstants.java

```diff
@@ -19,15 +19,21 @@
     public static final String INDEX_DIR = "../plan";
     public static final String FILES_DIR = "../plan_files";
     public static final String FILE_SUFFIX = ".ser";
```

Comment:
I have not refactored this class to be used only with JSON objects, yet. All of these already existed.